### PR TITLE
fix(card): core, angularjs and angular fixes

### PR DESCRIPTION
**Pull request by mathsusan**
BREAKING CHANGE:

angularng cards will be broken

## Description

fix the core css to make sure cards css for angularng don't conflict

## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [x ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ x] I have read the **CONTRIBUTING** document.
- [x ] I have added tests to cover my changes.
- [x ] All new and existing tests passed.

### CODE REVIEW

File: core/scss/ng/card.scss [hunk 1]
```diff
@@ -18,23 +18,23 @@
       }
     }
 
-    .md-card-layout {
-      &.md-card-block {
+    .md-ng1-card-layout {
+      &.md-ng1-card-block {
```
Comment (bfbiggs) [inline core/scss/ng/card.scss]: I think we only need to add ng1 to the top levels. This will reduce adoption time for html users as well.

Comment (daanishnasir) [reply core/scss/ng/card.scss]: Good idea

.md-ng1-card {
   &--block

**Comment by mathsusan** [comment]
I don't think we want to change the other cards until this is in?  Plus you said to separate the two commits

File: core/scss/ng/card.scss (hunk 1 above)
Comment (mathsusan) [reply core/scss/ng/card.scss]: made the change on the top .. not touching the rest as I have no way to test this change

**Comment by bfbiggs** [comment]
Couple Issues:
- initial commit should be fix(card) because in scss/ng the file is singular
- core examples need to be updated to remove the v2
- angularjs cards component needs to be updated to the new class

File: angularjs/src/lib/components/cardnumber/cardnumber.component.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@ export class CardNumberCtrl implements ng.IComponentController {
 export class CardNumberComponent implements ng.IComponentOptions {
   public controller = CardNumberCtrl;
   public template = `
-  <div class="md-card-number">
+  <div class="md-ng1-card-number">
```
Comment (bfbiggs) [inline angularjs/src/lib/components/cardnumber/cardnumber.component.ts]: No need to change this since it wasn't modified in the scss/ng/card-number.scss.

**Pull request merged**